frontend/public/components/console-notifier.tsx

```diff
@@ -16,7 +16,9 @@ const ConsoleNotifier_: React.FC<ConsoleNotifierProps> = ({ obj, location }) =>
     <React.Fragment>
       {_.map(_.get(obj, 'data'), (notification) =>
         notification.spec.location === location ||
-        notification.spec.location === 'BannerTopBottom' ? (
+        notification.spec.location === 'BannerTopBottom' ||
+        location === 'BannerTop' && notification.spec.location === '' ||
```

Comment:
This is a tad verbose but relatively easy to read.

Reply:
If we default the value, I think the logic gets simpler. (I also think the early return is easier to understand than the ternary even if there are two return statements.)

```tsx
const notificationLocation = notification.spec.location || 'BannerTop';
if (notificationLocation !== location && notificationLocation !== 'BannerTopBottom') {
  return null;
}
return <div>...</div>;
```

Reply:
looks good to me. will update.